melos.yaml

```
@@ -12,13 +12,19 @@ scripts:
     run: melos exec --file-exists="l10n.yaml" -- "flutter gen-l10n && melos bootstrap"
 
   build-tasks-apk:
```

```suggestion
  build-tasks-apk-debug:
```
The same goes for the other scripts as well.

---

No, I would like to define the default build configuration as `debug` as it's not possible to build a release on your own without using a proper key. If we need a release build command in `melos` later on, we should name it `build-tasks-apk-release`. But very likely we don't need a separate melos command for that.

---

> No, I would like to define the default build configuration as debug as it's not possible to build a release on your own without using a proper key. 

@use-to got it. Good idea. But what keeps us from being more specific in the name of the configuration? We are doing the same in the CI as well.

---

No, it's different to the CI configuration, as the CI will also build a release version later on. So we need to distinguish between debug and release. Why should I add space for a configuration a developer does not need or intent to use. Can we just leave it as it is and move this discussion? I don't see any more valid arguments on both sides.

---

_moved_